common/test/acceptance/pages/lms/progress.py

```diff
@@ -18,6 +18,35 @@ def is_browser_on_page(self):
         )
         return is_present
 
+    def get_element_padding(self, selector):
```

Comment:
This seems like a useful tool for other pages to leverage as well -- would it be possible to move into a parent object or utility package instead of the progress page object?